### ISSUE TITLE
CommandLinkDialog unusable.

### ISSUE THREAD

**Issue by b0c1** [controlsfx/controlsfx#1397]
Hi guys,
 
How can I handle more than two buttons with CommandLinksButtonType ?
I mean, the constructor which using ButtonType is private and the other public constructor can create OK_DONE, and OTHER button data.
Why private the other constructors?
b0c1

**Comment by eugener** [comment]
If you take a look at the [source code ](https://github.com/pkkannStation/Andraligo-upload-tool/blob/a2e18466baf8accf7e20ce601e795b740220b168/lib/ControlsFX/controlsfx%20source/controlsfx/src/main/java/org/controlsfx/dialog/CommandLinksDialog.java#L60), you will see that the first 3 constructors actually create `CommandLinksButonType` using custom `ButtonType`.

**Comment by b0c1** [comment]
@eugener no. The first 3 constructor create a ButtonType (it's not custom, it's `javafx.scene.control.ButtonType`) with pre-filled text, and button data. You can't use your own (custom) ButtonType because the constructor is private:
```java
private CommandLinksButtonType(ButtonType buttonType)
```

1. Why private are the constructors where you can set custom button type, text, and graphics?
2. If you create a CommonLinksDialog, the only way to use it correctly is if you extend the `CommandLinkButtonType`, you call some (not private ofc) constructor and override the getters with your own... so `CommandLinkButtonType` is great while you make an example where the button does nothing (not need to know which button pressed)

You said it's working correctly, show me an example with only 3 buttons and each button click to print out different text.
